maykin_2fa/templates/maykin_2fa/recovery_token.html

```diff
@@ -10,6 +10,19 @@
             recovery codes to verify your identity.
         {% endblocktrans %}</p>
 
+        {% if form.errors %}
+            <div class="error-messages">
+                {% for field in form %}
+                    {% for error in field.errors %}
+                        <p class="error">{{ error }}</p>
```

Comment:
We can reuse the `errornote` class to give it some styling :)
```suggestion
                        <p class="errornote">{{ error }}</p>
```

![afbeelding](https://github.com/user-attachments/assets/1efafa67-5449-44fa-a593-55ab313f8f6c)